neonvm-runner/cmd/net.go

```diff
@@ -163,7 +163,7 @@ func defaultNetwork(logger *zap.Logger, cidr string, ports []vmv1.Port) (mac.MAC
 	// pass incoming traffic to .Guest.Spec.Ports into VM
 	var iptablesArgs []string
 	for _, port := range ports {
-		logger.Info(fmt.Sprintf("setup DNAT rule for incoming traffic to port %d", port.Port))
+		logger.Debug(fmt.Sprintf("setup DNAT rule for incoming traffic to port %d", port.Port))
```

Comment:
I don't think we have a way of setting neonvm-runner's log level. Should we just delete these?

Reply:
Even if we don't have a way to change log level yet, debug/trace logs can serve the same role as comments, so I think we can keep them.